deploy/charts/trust-manager/templates/certificate.yaml

```diff
@@ -24,6 +24,8 @@ spec:
   dnsNames:
   - "{{ include "trust-manager.name" . }}.{{ include "trust-manager.namespace" . }}.svc"
   secretName: {{ include "trust-manager.name" . }}-tls
+  secretTemplate:
+    {{- toYaml .Values.certificates.certManager.secretTemplate | nindent 4 }}
```

Comment:
Maybe add a conditional around this, so we avoid setting `secretTemplate` to an empty dict by default?

Reply:
Thanks! Added condition around secretTemplate and fixed value ref.